sktime/forecasting/base/_base.py

```diff
@@ -198,9 +199,16 @@ def predict(self, fh=None, X=None, return_pred_int=False, alpha=DEFAULT_ALPHA):
         y_pred_int : pd.DataFrame - only if return_pred_int=True
             Prediction intervals
         """
+        # handle inputs
         self.check_is_fitted()
         self._set_fh(fh)
 
+        if return_pred_int and not self.get_tag("capability:pred_int"):
+            raise NotImplementedError(
+                f"{self.__class__.__name__} does not have the capability to return "
```

Comment:
This may be misleading too now, since some estimators have the capability in principle but it's simply not implemented yet. We may also want to default to bootstrapping or estimator agnostic way of generating prediction intervals similar to R's `forecast` package.

Reply:
>This may be misleading too now, since some estimators have the capability in principle but it's simply not implemented yet. 

Imo, there is nothing misleading since the error message refers to the `sktime` estimator right now, and not an ideal version of it which may or may not have the capability and on which we may or may not disagree what the natural "extension" to the capability is. I.e., my error message is factual and descriptive of the is-state, whereas the reference point for yours is unclear.

I don't mind putting your error message there since it's a very minor thing and most users won't see it. Do you have a strong opinion on this?

Reply:
after some thinking, I've updated the message to sth that should satisfy both our requirements?

The message refers to the is-state, and encourages users to open an issue if they think the capability should be there.

How about this?

Reply:
The confusion comes from conflating things which are simply not implemented and capabilities of an estimator.  Capability is not a common concept that users will be familiar with. When I hear it, I think of what an estimator can do in principle, not about what has or has not been implemented. 

But let's merge this as it is.